tests/resource/aws_imagebuilder_image_pipeline: Schedule Expression Test Failures

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### Description

The two `aws_imagebuilder_image_pipeline` resource acceptance tests that configure schedule expressions are currently failing:

```
=== CONT  TestAccAwsImageBuilderImagePipeline_Schedule_PipelineExecutionStartCondition
resource_aws_imagebuilder_image_pipeline_test.go:378: Step 1/3 error: Error running apply:
Error: error creating Image Builder Image Pipeline: InvalidParameterValueException: The value supplied for parameter 'schedule.scheduleExpression' is not valid. Invalid schedule expression. Details: Invalid CRON expression: error at position 9, details: Unexpected end of expression.
--- FAIL: TestAccAwsImageBuilderImagePipeline_Schedule_PipelineExecutionStartCondition (47.57s)

=== CONT  TestAccAwsImageBuilderImagePipeline_Schedule_ScheduleExpression
resource_aws_imagebuilder_image_pipeline_test.go:412: Step 1/3 error: Error running apply:
Error: error creating Image Builder Image Pipeline: InvalidParameterValueException: The value supplied for parameter 'schedule.scheduleExpression' is not valid. Invalid schedule expression. Details: Invalid CRON expression: error at position 9, details: Unexpected end of expression.
--- FAIL: TestAccAwsImageBuilderImagePipeline_Schedule_ScheduleExpression (43.82s)
```

The configuration needs to be updated, but at first glance the cron expressions that previously worked fine do match the AWS documentation.

### Affected Resource(s)

* aws_imagebuilder_image_pipeline

### References

* https://docs.aws.amazon.com/imagebuilder/latest/userguide/cron-expressions.html

Hmmm trying that in the acceptance testing still showed failures for me in the Commercial main and standalone accounts (just to verify its nothing weird with preview access) -- the error shifts from 9th character to 10th:

```
=== CONT  TestAccAwsImageBuilderImagePipeline_Schedule_PipelineExecutionStartCondition
    resource_aws_imagebuilder_image_pipeline_test.go:378: Step 1/3 error: Error running apply:
        Error: error creating Image Builder Image Pipeline: InvalidParameterValueException: The value supplied for parameter 'schedule.scheduleExpression' is not valid. Invalid schedule expression. Details: Invalid CRON expression: error at position 10, details: Unexpected end of expression.


--- FAIL: TestAccAwsImageBuilderImagePipeline_Schedule_PipelineExecutionStartCondition (20.42s)
=== CONT  TestAccAwsImageBuilderImagePipeline_Schedule_ScheduleExpression
    resource_aws_imagebuilder_image_pipeline_test.go:412: Step 1/3 error: Error running apply:
        Error: error creating Image Builder Image Pipeline: InvalidParameterValueException: The value supplied for parameter 'schedule.scheduleExpression' is not valid. Invalid schedule expression. Details: Invalid CRON expression: error at position 10, details: Unexpected end of expression.


--- FAIL: TestAccAwsImageBuilderImagePipeline_Schedule_ScheduleExpression (24.54s)
```

Do you have ideas how you got it to work?

I didn't mention it above, but the current cron syntax passes in AWS GovCloud (US). I'll keep fiddling by trying the [extended CloudWatch schedule expression syntax](https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html) with its year field to see if maybe that is something new but not documented. Otherwise I'll just raise an AWS Support case.

--- a/aws/resource_aws_imagebuilder_image_pipeline_test.go
+++ b/aws/resource_aws_imagebuilder_image_pipeline_test.go
@@ -415,11 +415,11 @@ func TestAccAwsImageBuilderImagePipeline_Schedule_ScheduleExpression(t *testing.
 		CheckDestroy: testAccCheckAwsImageBuilderImagePipelineDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccAwsImageBuilderImagePipelineConfigScheduleScheduleExpression(rName, "cron(1 0 * * *)"),
+				Config: testAccAwsImageBuilderImagePipelineConfigScheduleScheduleExpression(rName, "cron(1 0 * * ? *)"),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckAwsImageBuilderImagePipelineExists(resourceName),
 					resource.TestCheckResourceAttr(resourceName, "schedule.#", "1"),
-					resource.TestCheckResourceAttr(resourceName, "schedule.0.schedule_expression", "cron(1 0 * * *)"),
+					resource.TestCheckResourceAttr(resourceName, "schedule.0.schedule_expression", "cron(1 0 * * ? *)"),
 				),
 			},
 			{
@@ -428,11 +428,11 @@ func TestAccAwsImageBuilderImagePipeline_Schedule_ScheduleExpression(t *testing.
 				ImportStateVerify: true,
 			},
 			{
-				Config: testAccAwsImageBuilderImagePipelineConfigScheduleScheduleExpression(rName, "cron(2 0 * * *)"),
+				Config: testAccAwsImageBuilderImagePipelineConfigScheduleScheduleExpression(rName, "cron(2 0 * * ? *)"),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckAwsImageBuilderImagePipelineExists(resourceName),
 					resource.TestCheckResourceAttr(resourceName, "schedule.#", "1"),
-					resource.TestCheckResourceAttr(resourceName, "schedule.0.schedule_expression", "cron(2 0 * * *)"),
+					resource.TestCheckResourceAttr(resourceName, "schedule.0.schedule_expression", "cron(2 0 * * ? *)"),
 				),
 			},
 		},
@@ -808,7 +808,7 @@ resource "aws_imagebuilder_image_pipeline" "test" {
 
   schedule {
     pipeline_execution_start_condition = %[2]q
-    schedule_expression                = "cron(0 0 * * *)"
+    schedule_expression                = "cron(0 0 * * ? *)"
   }
 }
 `, rName, pipelineExecutionStartCondition))

--- a/website/docs/r/imagebuilder_image_pipeline.html.markdown
+++ b/website/docs/r/imagebuilder_image_pipeline.html.markdown
@@ -19,7 +19,7 @@ resource "aws_imagebuilder_image_pipeline" "example" {
   name                             = "example"
 
   schedule {
-    schedule_expression = "cron(0 0 * * *)"
+    schedule_expression = "cron(0 0 * * ? *)"
   }
 }
 ```
@@ -53,7 +53,7 @@ The following arguments are optional:
 
 The following arguments are required:
 
-* `schedule_expression` - (Required) Cron expression of how often the pipeline start condition is evaluated. For example, `cron(0 0 * * *)` is evaluated every day at midnight UTC.
+* `schedule_expression` - (Required) Cron expression of how often the pipeline start condition is evaluated. For example, `cron(0 0 * * ? *)` is evaluated every day at midnight UTC. Configurations using the five field syntax that was previously accepted by the API, such as `cron(0 0 * * *)`, must be updated to the six field syntax. For more information, see the [Image Builder User Guide](https://docs.aws.amazon.com/imagebuilder/latest/userguide/cron-expressions.html).
 
 The following arguments are optional:
 

